lib/plottr_components/src/components/images/ImagePicker.js

```diff
@@ -128,8 +128,11 @@ const ImagePickerConnector = (connector) => {
     }
 
     deleteImage = () => {
+      const { chooseImage, actions } = this.props
+
       if (this.state.deleting) {
-        this.props.actions.deleteImage(this.state.selectedId)
+        actions.deleteImage(this.state.selectedId)
+        chooseImage(null)
```

Comment:
This part doesn’t make sense to me.  You should be able to delete an image in the picker without choosing no image.  What if you want to delete a bunch of images and then pick an image?